aptos-move/aptos-vm/src/sharded_block_executor/sharded_executor_service.rs

```diff
@@ -23,8 +24,29 @@ use aptos_types::{
 use aptos_vm_logging::disable_speculative_logging;
 use futures::{channel::oneshot, executor::block_on};
 use move_core_types::vm_status::VMStatus;
+use once_cell::sync::Lazy;
 use std::sync::Arc;
 
+pub static APTOS_SHARDED_EXECUTOR_EXECUTE_BLOCK_SECONDS: Lazy<Histogram> = Lazy::new(|| {
```

Comment:
nit: it's sorta our convention to host these counters in a separate file (counters.rs or metrics.rs)
and nit: shall we use `register_histogram_vec` to make our lifes easier in most cases? (both when coding and on the dashboards)
for example https://github.com/aptos-labs/aptos-core/blob/3ba06ec9f1aab0206f7e0d822e3531b87a218fe9/experimental/execution/ptx-executor/src/metrics.rs#L9-L33

Reply:
+1 to use single metric, unless you need a different bucket

Reply:
@msmouse we are already using quite a few stats with register_histogram, say like APTOS_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS.

Do you still prefer using `register_histogram_vec` only ?

> pub static APTOS_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS: Lazy<Histogram> = Lazy::new(|| {
    register_histogram!(
        // metric name
        "aptos_executor_vm_execute_block_seconds",
        // metric description
        "The time spent in seconds of vm block execution in Aptos executor",
        exponential_buckets(/*start=*/ 1e-3, /*factor=*/ 2.0, /*count=*/ 20).unwrap(),
    )
    .unwrap()
});

Reply:
yeah.. I still think it's nice to just use https://github.com/aptos-labs/aptos-core/blob/7329c16e7d336f5f1708310d7c09d237429d8bfb/aptos-move/aptos-vm/src/counters.rs#L142
or add a `SHARDED_EXECUTOR_TIMER` to group things up